plat/unix/update_tags.sh

```diff
@@ -99,7 +99,13 @@ if [ -f "$TAGS_FILE" ]; then
     if [ "$UPDATED_SOURCE" != "" ]; then
         echo "Removing references to: $UPDATED_SOURCE"
         tab="	"
-        cmd="grep --text -Ev '^[^$tab]+$tab$UPDATED_SOURCE$tab' '$TAGS_FILE' > '$TAGS_FILE.temp'"
+        # use ripgrep if available
+        if command -v rg > /dev/null 2>&1; then
+            GREP_CMD="rg --noconfig"
```

Comment:
I don't know why you changed my suggested `--no-config` to `--noconfig`. At least in ripgrep 13.0 it's `--no-config`:

>        --no-config
>           Never read configuration files. When this flag is present, ripgrep will not respect the RIPGREP_CONFIG_PATH environment variable.

(source: my local man page)

Also see the [zsh completion function](https://github.com/BurntSushi/ripgrep/blob/962d47e6a1208cf2187cd34c2a7f6cf32e2a4903/complete/_rg#L316) or [app.rs](https://github.com/BurntSushi/ripgrep/blob/962d47e6a1208cf2187cd34c2a7f6cf32e2a4903/crates/core/app.rs#L1991).

Reply:
my bad. its a typo. thanks for catching it.

Reply:
Now it looks good to me. But I'm just a user, not a maintainer of this project.